pkg/util/applier/direct.go

```diff
@@ -132,8 +131,8 @@ func newOptions(flags *apply.ApplyFlags, namespace string) (*apply.ApplyOptions,
 		DynamicClient:    dynamicClient,
 
 		IOStreams:         flags.IOStreams,
-		VisitedUids:       sets.NewString(),
-		VisitedNamespaces: sets.NewString(),
+		VisitedUids:       sets.New[types.UID](),
+		VisitedNamespaces: sets.New[string](),
```

Comment:
FYI for future git blamers :)
Changes here^ were made manually, so now it diverges from the original source from
https://github.com/kubernetes-sigs/kubebuilder-declarative-pattern/tree/0867fae819470ae478f2a90df9d943f5b7ee0b4f/pkg/patterns/declarative/pkg/applier/direct.go